voting/src/error.rs

```diff
@@ -0,0 +1,53 @@
+use cosmwasm_std::{CanonicalAddr, StdError};
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ContractError {
+    #[error("{0}")]
+    Std(#[from] StdError),
+
+    #[error("insufficient funds sent")]
+    InsufficientFunds {},
```

Comment:
```suggestion
    InsufficientFundsSent {},
```

Otherwise it sounds like the balance is not enought